### DISCUSSION TITLE
How do you flush the data,and remove old data from the db?

### DISCUSSION THREAD

**Discussion by J-Programmer** [imartinez/privateGPT#841] [Q&A]
I deleted the default source data, 'state_of_the_union.txt,' and added my own source document. However, the model continues to reference both documents when responding to my query. How can I flush the data and remove the old data from the database?

**Comment by mechanicmuthu** [comment]
Change database name in dot env file or delete existing database folder in the project folder.

**Comment by J-Programmer** [comment]
Delete the database folder and re-ingest with the new source document?  Is the DB folder created during ingest.py?

**Reply by J-Programmer** [reply]
I just read the ingest.py code, I'll try the suggested above. Thanks!

**Reply by ghost** [reply]
This hasn't worked for me - deleting the folder and its contents and adding an empty folder and filling it does not remove the data from the previous ingest. There must be a way to purge previously ingested data as the model gets confused very quickly.
